### PR TITLE
Export users csv

### PR DESCRIPTION
**Pull request by danielkesselberg**

### CODE REVIEW

File: protected/humhub/modules/admin/controllers/PendingRegistrationsController.php [hunk 1]
```diff
@@ -56,103 +40,73 @@ public function getAccessRules()
...
-            ['language'],
-            ['source'],
-            ['created_at', 'type' => 'datetime'],
+        return [
+            'id',
+            'user_originator_id',
+            'space_invite_id',
+            'email',
+            'source',
+            'token',
```
Comment (githubjeka) [inline protected/humhub/modules/admin/controllers/PendingRegistrationsController.php]: For what token?

File: protected/humhub/modules/admin/controllers/PendingRegistrationsController.php [hunk 2]
```diff
@@ -56,103 +40,73 @@ public function getAccessRules()
...
+            'resultConfig' => [
+                'fileBaseName' => 'humhub_user',
+                'writerType' => $format,
+            ],
+        ]);
+
+        return $exporter->export()->send();
     }
 
-    public function createCVS($dataProvider, ActiveRecord $model, $format = null)
```
Comment (githubjeka) [inline protected/humhub/modules/admin/controllers/PendingRegistrationsController.php]: BC break

File: protected/humhub/modules/admin/controllers/PendingRegistrationsController.php [hunk 3]
```diff
@@ -56,103 +40,73 @@ public function getAccessRules()
         ];
     }
 
-    public function actionIndex($export = false, $format = null)
+    public function actionIndex()
```
Comment (githubjeka) [inline protected/humhub/modules/admin/controllers/PendingRegistrationsController.php]: BC break

File: protected/humhub/modules/admin/controllers/PendingRegistrationsController.php (hunk 2 above)
Comment (danielkesselberg) [reply protected/humhub/modules/admin/controllers/PendingRegistrationsController.php]: Thank you! We should add a note to changelog that createCVS is removed.

File: protected/humhub/modules/admin/controllers/PendingRegistrationsController.php (hunk 1 above)
Comment (danielkesselberg) [reply protected/humhub/modules/admin/controllers/PendingRegistrationsController.php]: Thank you. Good point.

**Pull request merged**